documentation/ame-guide/modules/ROOT/pages/generate/generate-aasx.adoc

```diff
@@ -1,15 +1,15 @@
 = Generate AASX/XML
 
-To generate an AASX/XML files from the Aspect Model, proceed as follows:
+To generate AASX/XML files from an Aspect Model, proceed as follows:
 
-* Click the *Generate* on xref:getting-started/ui-overview.adoc#menu-generate[menu]. +
-A menu offers one option:
-
-** *AASX/XML* -- continue with this one
+* Click the xref:getting-started/ui-overview.adoc#menu-generate[Generate menu] and select *AASX/XML*.
 +
-image::generate-aasx.png[Generate AASX/XML,width=100%,align="left"]
+image:generate-aasx.png[Generate AASX/XML, width=15%]
+
 * Select *AASX* or *XML* as the desired file format.
```

Comment:
Maybe add here "in the drop-down" or similiar? 
![image](https://github.com/eclipse-esmf/esmf-aspect-model-editor/assets/124154190/38df9203-495a-46a5-bbd0-12b228fce358)
Without having a screenshot I could not "simulate" that step. 😊
I am not aware of the rest of this documentation and if we also format the file types here in bold?